### ISSUE TITLE
Write 4.x documentation

### ISSUE THREAD

**Issue by virustotalop** [ClubObsidian/DynamicGui#100]
The documentation for 3.x was not every fully written and documentation for [4.x is currently being ported.](https://github.com/dynamicgui/documentation) Before we start using 4.x fully in production everything should be document as the current treeing functionality is confusing and has a greater learning curve than the previous version.

**Comment by virustotalop** [comment]
Superseded by #216
